src/main/java/redis/clients/jedis/commands/JCommands.java

```diff
@@ -0,0 +1,327 @@
+package redis.clients.jedis.commands;
+
+import redis.clients.jedis.BitPosParams;
+import redis.clients.jedis.GeoCoordinate;
+import redis.clients.jedis.GeoRadiusResponse;
+import redis.clients.jedis.GeoUnit;
+import redis.clients.jedis.ListPosition;
+import redis.clients.jedis.ScanParams;
+import redis.clients.jedis.ScanResult;
+import redis.clients.jedis.SortingParams;
+import redis.clients.jedis.Tuple;
+import redis.clients.jedis.params.GeoRadiusParam;
+import redis.clients.jedis.params.SetParams;
+import redis.clients.jedis.params.ZAddParams;
+import redis.clients.jedis.params.ZIncrByParams;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author Petar Tahchiev
+ * @since 2.0.1
+ */
+public interface JCommands {
```

Comment:
I think we need a better naming

Reply:
Agree

Reply:
I have renamed `JCommands` to `JedisBaseCommands`